RELEASE-NOTES.txt

```diff
@@ -1,3 +1,10 @@
+1.1.1
+=====
+* Updated wp-now to fix missing curl and webp issues
```

Comment:
Could you add PR links where possible, like we did for previous versions?

Reply:
Added in https://github.com/Automattic/studio/pull/523/commits/5bfc33f9dcd71909e0f514eb2dd6df6823c6379a